src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -47,26 +47,13 @@ import { SavedObjectsSerializer } from './serialization';
 
 /**
  * Saved Objects is Kibana's data persistence mechanism allowing plugins to
- * use Elasticsearch for storing and querying state. The
- * SavedObjectsServiceSetup API exposes methods for creating and registering
- * Saved Object client wrappers.
+ * use Elasticsearch for storing and querying state. The SavedObjectsServiceSetup API exposes methods
+ * for registering Saved Object types and creating and registering Saved Object client wrappers and factories
  *
  * @remarks
- * Note: The Saved Object setup API's should only be used for creating and
- * registering client wrappers. Constructing a Saved Objects client or
- * repository for use within your own plugin won't have any of the registered
- * wrappers applied and is considered an anti-pattern. Use the Saved Objects
- * client from the
- * {@link SavedObjectsServiceStart | SavedObjectsServiceStart#getScopedClient }
```

Comment:
Removed outdated comment, as it's no longer possible to create clients or repository from the setup contract